src/routes/routeConstants.ts

```diff
@@ -0,0 +1,3 @@
+export const RoutesPath = {
```

Comment:
what convention will we be following for constants ?  we have upper snake case in recent project.